mobile/src/main/java/ch/epfl/sdp/mobile/ui/prepare_game/GameTypeChoiceButtons.kt

```diff
@@ -0,0 +1,71 @@
+package ch.epfl.sdp.mobile.ui.prepare_game
+
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.*
+import androidx.compose.material.ButtonDefaults.buttonColors
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.dp
+import ch.epfl.sdp.mobile.state.LocalLocalizedStrings
+import ch.epfl.sdp.mobile.ui.Check
+import ch.epfl.sdp.mobile.ui.PawniesColors
+import ch.epfl.sdp.mobile.ui.PawniesIcons
+
+@Composable
+fun GameTypeChoiceButtons(
+    gameType: GameType,
+    onNewOnlineGame: () -> Unit,
+    onNewLocalGame: () -> Unit,
+    modifier: Modifier = Modifier
+) {
+  val strings = LocalLocalizedStrings.current
+  Column(
```

Comment:
You may want to make sure the children of this column fill the parent max width. There are multiple ways to do this, the simplest (yet not necessarily correct, but good enough for now) being to add `Modifier.fillMaxWidth()` to the children `Button`.

Reply:
👍